trainings/sourcegraph-101/slack-follow-up.md

```diff
@@ -0,0 +1,23 @@
+< DATE >.
```

Comment:
```suggestion
Thanks so much for attending our Sourcegraph 101 training! As we mentioned on the call, we'll be following up with a Sourcegraph 102 training for advanced tips and tricks on `< DATE >`.
```

@emchap did you mean to remove the intro part of this too? Disregard this suggestion if so, sorry!

Reply:
@IAmHughes Nope! Thanks for the catch.